ablog/start.py

```diff
@@ -216,7 +216,7 @@ def w(t, ls=80):
 ]
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ["{dot}templates", ablog.get_html_templates_path()]
+templates_path = [ablog.get_html_templates_path(), "{dot}templates"]
```

Comment:
Same here, should be first.

Reply:
Users don't need to do this, given that the event handler adds this path already for them.

Reply:
Right, I will remove this from the setup script.

Reply:
Actually, we might want to add a check to reject configurations that do this. If users are doing this, they'll be able to break the way that the suggestion I've made for handling #143 and #125 works in a manner that's gonna be difficult for the user to diagnose (https://github.com/sunpy/ablog/pull/144#discussion_r927425327).

Reply:
To elaborate on what I mean, here's the situation elaborated upon:

- Furo adds support for Ablog and declares `inject_templates_after_theme = true` -- to say that the theme's templates should take precedence over Ablog's templates (allowing Furo to have a custom `page.html` and `ablog/catalog.html` for example).
- The above suggested code will make Ablog inject its own templates *after* the theme templates.

If the user does nothing more, everything will work just fine. However, if the user sets this in their `conf.py` file, the Ablog templates will also be added *before* the theme templates (since user templates are injected before theme templates). This would mean that Ablog's own templates would take precedence over the theme's templates. See bullet 1, which expects the exact opposite.

The most annoying part of this for the user is that the build won't fail, since the templates will still likely render correctly -- they'll be getting picked up from the wrong place and cause improper markup to be generated in the final rendered templates!